src/platforms/unreal/setup-crashreport.mdx

```diff
@@ -81,6 +78,59 @@ is enough.
 
 </Alert>
 
+## Configure the Crash Reporter Attributes {#configure-attributes}
+
+Basic attributes can be reconfigured by providing special game data to the
```

Comment:
Can we achieve the same through the API in the SDK itself? If so, we could point that out here.